### PR TITLE
Fix alignment issue in AdditionalFees component

### PR DESCRIPTION
**Pull request by lisunshiny**
Fix bloom-housing/bloom#3220 by removing some unnecessary margin classes (maybe cruft from a previous world where this component had application fee & deposit side by side?).

I haven't used GitHub properly in years, plz advise if I miss anything!

### CODE REVIEW

**Comment by lisunshiny** [comment]
@slowbot gotcha. Question -- would you recommend still using the InfoCard component here, or would you favor Card (https://storybook.bloom.exygy.dev/?path=/docs/blocks-card--text-content), or a different component altogether? Card has some nice stuff like a footer, which would be useful here.

I figure if the job is to match the AdditionalFees to the spec the best way is to use the component library directly instead of these divs/classes, but am not sure which one. WDYT?

**Comment by lisunshiny** [comment]
@slowbot  -- actually I decided that bringing the whole component up to spec is outside of the scope of this change, but I did make some changes to bring the two columns back. imo this is a strict improvement. lmk!

File: src/page_components/listing/AdditionalFees.tsx
Comment (emilyjablonski) [inline src/page_components/listing/AdditionalFees.tsx:2]: Let's import these from their individual files like `import { GridSection } from "../../sections/GridSection"`

**Review by emilyjablonski — APPROVED** [review]
Great q about which component to pull in - we definitely prefer an existing component in use here so this is a great update! Agreed that getting us fully up to spec is a separate issue and this is a good improvement. After one more approval & an update to the imports this is ready to merge :)

File: src/page_components/listing/AdditionalFees.tsx
Comment (lisunshiny) [reply src/page_components/listing/AdditionalFees.tsx]: done

**Comment by lisunshiny** [comment]
Thanks for the review @emilyjablonski! Updated with your change and also fixed the double margin UI annoyance in a better way

Do you know who I should tag for review number 2 (and more generally, who I should tag to review CLs to bloom repos?)

Lastly I squash commit right? If I also missed any of the CL review best practices lmk.

**Review by ColinBuyck — APPROVED** [review]
This all LGTM 🌲 Thank you for making these improvements! In terms of merging, squash commit is best. I'll have to defer to @emilyjablonski for the CL best practices portion of your question though.

**Comment by emilyjablonski** [comment]
Since this has two approvals and I'm not sure we've set up git permissions yet, I'll go ahead and merge this one - thank you!

**Comment by lisunshiny** [comment]
SG! Feel free to merge whenever, you're right I don't have access 😓. Lots of useful info in https://github.com/bloom-housing/ui-components/pull/8#issuecomment-1396300315 so I shared it with the team as well so they know, and sg. re slack channel. Thanks for the speedy reviews

**Pull request merged**
